daemon/src/cli.rs

```diff
@@ -39,6 +39,9 @@ pub enum Commands {
         /// keep Magic Wormhole out of the loop.
         #[arg(long)]
         no_join_code: bool,
+        /// Use an alternative magic wormhole rendezvous url
```

Comment:
For consistency, could we write "Magic Wormhole" capitalized here?

Reply:
Ok, while we're here, and this is not on you, @martengooz, at all: We could make it consistent all the way through. I just searched for a lowercase "magic wormhole" in the codebase:
- https://github.com/teamtype/teamtype/blob/20270b5aee5565c3b9283fafc1bdca1181c60181/docs/teamtype-cli-ui.md?plain=1#L19 this is even completely inconsistent, because it looks like a command but is missing the `-` in it. Let's just put "Magic Wormhole" there instead of "`magic wormhole`".
- https://github.com/teamtype/teamtype/blob/20270b5aee5565c3b9283fafc1bdca1181c60181/daemon/src/wormhole.rs#L20

I thought it's tedious that you have to fix our legacy so I just pushed it myself in [3996298](https://github.com/teamtype/teamtype/pull/446/commits/3996298b072c561630cee12673dadef09443fadb) :)